src/gan/gen_pyt/components/vocab.py

```diff
@@ -21,14 +23,10 @@ def __init__(self, filename=None, data=None, lower=False):
         self.add('<eos>')
 
     def __getitem__(self, item):
-            return self.labelToIdx.get(item, self.unk)
+        return self.labelToIdx.get(item, self.unk)
 
     def __contains__(self, item):
-            return item in self.labelToIdx
-
-    @property
-    def size(self):
```

Comment:
Do we not use `Vocab.size()` anywhere? We might not, just want to check.

Reply:
Good question. 

If we do, I don't think it would work. The problem is that we overwrite the `size` property by declaring a size method immediately below:

https://github.com/GallupGovt/multivac/blob/d4484a517a9bed05265d6613391a557fc7b51291/src/gan/gen_pyt/components/vocab.py#L63-L64

You can't have both a property and a method with the same name. Whichever one is defined last will be used:


```
In [1]: class Foo: 
   ...:     
   ...:     @property 
   ...:     def size(self): 
   ...:         return 'self-property' 
   ...:     def size(self): 
   ...:         return 'self-method' 
   ...:                                                                                                                                                  

In [2]: foo = Foo()                                                                                                                                      

In [3]: foo.size                                                                                                                                         
Out[3]: <bound method Foo.size of <__main__.Foo object at 0x10e714790>>

In [4]: foo.size()                                                                                                                                       
Out[4]: 'self-method'

In [5]: class Foo: 
   ...:     
   ...:     def size(self): 
   ...:         return 'self-method' 
   ...:     @property 
   ...:     def size(self): 
   ...:         return 'self-property' 
   ...:          
   ...:                                                                                                                                                  

In [6]: foo = Foo()                                                                                                                                      

In [7]: foo.size                                                                                                                                         
Out[7]: 'self-property'

In [8]: foo.size()                                                                                                                                       
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-8-261b90f7afe4> in <module>
----> 1 foo.size()

TypeError: 'str' object is not callable